.circleci/config.yml

```diff
@@ -3,15 +3,39 @@ jobs:
   build:
     working_directory: ~/repo
     docker:
-      - image: honeycombio/cci-go-yq:0.0.8
+      - image: cimg/go:1.19.4
     steps:
       - checkout
       - restore_cache:
           keys:
             - builder-config-{{ checksum "builder-config.yaml" }}
+      - run:
+          name: install dependencies
+          command: |
+            sudo apt-get update -qq
+            sudo apt-get install -y python2  libbz2-dev  libreadline-dev libssl-dev openssl
+            curl https://pyenv.run | bash 
+            sudo rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+            export PYENV_ROOT=/home/circleci/.pyenv
+            export PATH=/home/circleci/.pyenv/shims:/home/circleci/.pyenv/bin:/home/circleci/.poetry/bin:$PATH
+            pyenv install 3.9.12
```

Comment:
Can we make these two use a single envvar, preferably without the patch specified?

Reply:
Done